### PR TITLE
feat(web): vite poc

### PR DESCRIPTION
**Pull request by scopsy**
TODO:
- Light Dark Mode not picking up proper styles 
- Fix fonts so that original `14px` fonts are converted to `0.875rem` instead of `1rem`

### CODE REVIEW

File: apps/web/index.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <!-- Google Tag Manager -->
-    <% if ( process.env.REACT_APP_NOVU_GTM_ID ) { %>
+<!--    <% if ( process.env.REACT_APP_NOVU_GTM_ID ) { %>
```
Comment (scopsy) [inline apps/web/index.html]: Need to find a replacement

File: apps/web/src/studio/components/workflows/step-editor/WorkflowStepEditorControlsPanel.tsx [hunk 2]
```diff
@@ -47,9 +49,13 @@ export const WorkflowStepEditorControlsPanel: FC<IWorkflowStepEditorControlsPane
     return Object.keys(step?.controls?.schema?.properties || step?.inputs?.schema?.properties || {}).length > 0;
   }, [step?.controls?.schema, step?.inputs?.schema]);
 
+  /*
   const handleOnChange = useDebouncedCallback(async (type: OnChangeType, data: any, id?: string) => {
     onChange(type, data, id);
   }, TYPING_DEBOUNCE_TIME_MS);
+*/
+
+  const handleOnChange = (type: OnChangeType, data: any, id?: string) => {};
```
Comment (scopsy) [inline apps/web/src/studio/components/workflows/step-editor/WorkflowStepEditorControlsPanel.tsx:67]: TODO: Fix

File: libs/novui/src/hooks/index.ts [hunk 3]
```diff
@@ -1 +1,3 @@
+export const useDebouncedCallback = function () {}; /*
 export { useDebouncedCallback } from '@mantine/hooks';
```
Comment (scopsy) [inline libs/novui/src/hooks/index.ts]: TODO: This is failing the build for some reason

File: apps/api/src/.env.test [hunk 4]
```diff
@@ -39,7 +39,7 @@ REDIS_CLUSTER_KEY_PREFIX=
 SYNC_PATH=http://127.0.0.1:3001
 API_ROOT_URL=http://127.0.0.1:3000
 DISABLE_USER_REGISTRATION=false
-PORT=1337
+PORT=1336
```
Comment (rifont) [inline apps/api/src/.env.test:39]: Aligning on port 1336 for api testing.

**Comment by rifont** [comment]
Closing as we are moving to the new Dashboard app.

**Pull request closed without merging**